app/client/components/transferAccount/TransferAccountList.tsx

```diff
@@ -361,6 +343,7 @@ class TransferAccountList extends React.Component<Props, ComponentState> {
     );
 
     const hasSelected = selectedRowKeys.length > 0;
+    const numberSelected = this.props.paginationOptions?.items ? allSelected ? this.props.paginationOptions?.items - unselectedRowKeys.length : selectedRowKeys.length : selectedRowKeys.length
```

Comment:
Changed from "All" to a number. This fixes the bug that shows "All Selected" when you select all and then _unselect_ a user.